languages.toml

```
@@ -2437,6 +2437,18 @@ language-server = { command = "nimlangserver" }
 name = "nim"
 source = { git = "https://github.com/aMOPel/tree-sitter-nim", rev = "240239b232550e431d67de250d1b5856209e7f06" }
 
+[[language]]
+name = "cabal"
+scope = "source.cabal"
+file-types = [ "cabal" ]
+roots = [ "cabal.project", "Setup.hs", "*.cabal" ]
```

Globs/wildcards aren't supported for `roots`

```suggestion
roots = ["cabal.project", "Setup.hs"]
```

---

Yeah that should be removed as well